cleverhans/utils.py

```diff
@@ -89,7 +89,7 @@ def other_classes(nb_classes, class_ind):
     return other_classes_list
 
 
-def to_categorical(y, num_classes=None):
```

Comment:
Making this argument mandatory is fine by me, but because this changes the API, can you add a 6-month deprecation warning like we usually do in the codebase, so that people get a chance to update their code before we remove the `=None` to make this argument mandatory?